src/main/java/cz/xtf/TestConfiguration.java

```diff
@@ -178,7 +179,10 @@ public static TestConfiguration get() {
 	public static String masterUrl() {
 		String result = get().readValue(DOMAIN);
 		if (StringUtils.isNotBlank(result)) {
-			result = "https://" + (result.equals(CDK_DOMAIN) ? CDK_IP : ("api." + result)) + ":8443";
+			result = "https://" + (result.equals(CDK_DOMAIN) ? CDK_IP : ("api." + result));
+			if(!openshiftOnline()){
```

Comment:
I don't believe that we should use xtf.config.domain property for OpenShift Online. Some other properties are delivered from this one (eg. ROUTE_DOMAIN, MAVEN_BASE_URL...) and could actually break a suite. MASTER_URL should be used with online testing